src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(String color, int firstLeg, int secondLeg, String name) {
+        setColor(color);
+        setName(name);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    public int getFirstLeg() {
+        return firstLeg;
+    }
+
+    public void setFirstLeg(int firstLeg) {
+        this.firstLeg = firstLeg;
+    }
+
+    public int getSecondLeg() {
+        return secondLeg;
+    }
+
+    public void setSecondLeg(int secondLeg) {
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public void draw() {
+
+    }
+
+    @Override
+    public double getArea() {

```

Comment:
The `getArea()` method should not simply call `super.getArea()`. Since this is a `RightTriangle`, you should implement the area calculation specific to a right triangle, which is `0.5 * base * height`.